Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+WORKDIR /clang-bind
+COPY . .
+
+RUN apt-get update && apt-get install -y python3-pip build-essential wget
+
+# Install libclang via development branch on https://apt.llvm.org/
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN echo 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-12 main' | tee -a /etc/apt/sources.list
+RUN echo 'deb-src http://apt.llvm.org/focal/ llvm-toolchain-focal-12 main' | tee -a /etc/apt/sources.list
```

Comment:
Why are the sourced needed? To install libclang-12-dev?

Reply:
To install a development branch basically. 
v12 is stable now, but we'll need this when we want to bump to v14 (currently developmental).

Reply:
Comment that please

Reply:
Also, don't use sources.list, instead create a new file in sources.list.d

Reply:
My bad, this method was to install stable branches that were not directly accessible via `apt-get install`, not for developmental branches. Hence, I've removed this block.